### PR TITLE
create legacy formula for ArangoDB 2.x

### PR DESCRIPTION
**Pull request by fceller**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-versions/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?
- [x] Does your submission adhere to Versions "Acceptable Formulae" [guidelines](https://github.com/Homebrew/homebrew-versions/blob/07b17dfb9f1/README.md#acceptable-formulae)?

-----

The new ArangoDB 3.x is not compatible (for database and Foxx apps) with the 2.x version.

### CODE REVIEW

File: arangodb2.rb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        </array>
+        <key>RunAtLoad</key>
+        <true/>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    assert_equal "it works!\n", shell_output("#{bin}/arangosh --javascript.execute-string \"require('@arangodb').print('it works!')\"")
```
Comment (MikeMcQuaid) [inline arangodb2.rb]: Test is failing: http://bot.brew.sh/job/Homebrew%20Versions%20Pull%20Requests/1460/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_homebrew_versions_arangodb2/

File: arangodb2.rb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  homepage "https://www.arangodb.com/"
+  url "https://www.arangodb.com/repositories/Source/ArangoDB-2.8.10.tar.gz"
+  sha256 "3f2adba7bb4fe5a93959c6929b984b053e66d3653144ed6bb1cdedd017c65050"
+
+  bottle do
+    revision 1
+    sha256 "f9699b28c53f744b7b4701890739d7a9369a6e0b144d486d7fee27ed4b4a9486" => :el_capitan
+    sha256 "7490e2e4026fca6442fce8c6f56d654deb45ba864ca6b7a73872d670b4475d49" => :yosemite
+    sha256 "60d59e09f4a440cffeb077f1de14784519223e6f25ccaaf6df692dc63fd6cac8" => :mavericks
+  end
```
Comment (MikeMcQuaid) [inline arangodb2.rb]: You can remove this bottle block; it'll get added when we pull.

Comment (fceller) [inline arangodb2.rb]: Done.

File: arangodb2.rb (hunk 1 above)
Comment (fceller) [inline arangodb2.rb]: Hopefully fixed. Need to test.

**Comment by fceller** [comment]
@BrewTestBot test this please

**Comment by fceller** [comment]
@ilovezfs can you merge this as well, now that arangodb 3.0 is available?

**Comment by DomT4** [comment]
Apologies, will re-queue the CI build.

**Comment by ilovezfs** [comment]
@fceller this repository is @DomT4's realm and and his favorite repository, so we'll need to wait on his approval.

**Comment by DomT4** [comment]
> and his favorite repository

It's definitely not 🙈.

**Comment by fceller** [comment]
@DomT4 @ilovezfs sorry, I was not aware of the realm separations :-)

**Comment by fceller** [comment]
@DomT4 @ilovezfs it is now all green

File: arangodb2.rb [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    s = <<-EOS.undent
+      Please note that clang and/or its standard library 7.0.0 has a severe
+      performance issue. Please consider using '--cc=gcc-5' when installing
+      if you are running on such a system.
+    EOS
+
+    s
+  end
+
+  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/arangodb2/sbin/arangod-2.8"
```
Comment (ilovezfs) [inline arangodb2.rb]: @fceller Doesn't this version require the config file for manual start?

Comment (fceller) [inline arangodb2.rb]: It should use the default configuration file. I did not really change much compared to the previous ArangoDB 2.8.9 version.

**Comment by ilovezfs** [comment]
:+1:

@DomT4 ?

**Comment by DomT4** [comment]
Seems fine. Thanks for working on this @ilovezfs & thank you again for your contributions to Homebrew @fceller. Merged in https://github.com/Homebrew/homebrew-versions/commit/4182f2609c7646c69c53f5cf590b117ceae6d262.

**Pull request closed without merging**